_data/contributors.yml

```
@@ -202,7 +202,7 @@
   contributor_type:
     - contributor
   packages-submitted: [""]
-  packages-reviewed: ["earthpy"]
+  packages-reviewed: ["pystiche"]
```

@NickleDave You seem to have replaced `"earthpy"` with `"pystiche"` for your reviewed packages. Was that intentional?

```suggestion
  packages-reviewed: ["earthpy", "pystiche"]
```